operator/v1/types_network.go

```diff
@@ -507,6 +507,65 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// ipv4 allows users to configure IP settings for IPv4 connections. When omitted, this means no opinion and the defualt
+	// configuration is used. Check individual members fields within ipv4 for details of default values.
+	// +optional
+	IPv4 *IPv4GatewayConfig `json:"ipv4,omitempty"`
+	// ipv6 allows users to configure IP settings for IPv6 connections. When omitted, this means no opinion and the defualt
+	// configuration is used. Check individual members fields within ipv6 for details of default values.
+	// +optional
+	IPv6 *IPv6GatewayConfig `json:"ipv6,omitempty"`
+}
+
+type IPv4GatewayConfig struct {
+	// v4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
```

Comment:
We don't need to duplicate that `v4` anymore, and we still don't mention that it's in CIDR format
```suggestion
	// internalMasqueradeSubnet contains the masquerade addresses in IPv4 CIDR format, used internally by
```